README.md

```diff
@@ -24,7 +31,14 @@ Download the latest release from [Releases](https://github.com/talklittle/tmsu-n
 
 **Alternatively, copy .so manually:**
 
+    # Debian
     sudo cp libtmsu-nautilus.so /usr/lib/nautilus/extensions-3.0/libtmsu-nautilus.so
+    # Fedora
+    sudo cp libtmsu-nautilus.so /usr/lib64/nautilus/extensions-3.0/libtmsu-nautilus.so
+
+    # Make sure the extension has the correct permission
+    sudo chmod u=rwxr,g=xr,o=x libtmsu-nautilus.so
+    sudo chmod g+x libtmsu-nautilus.so
```

Comment:
Is this second `chmod` command needed?

Reply:
Thinking the same thing. But in my case the first chmod doesn't add `x` to group, even after several trial. I will omit the second chmod, if someone had the similar issue. We can just point them here.